workshops/shared_clipboard/README.md

```diff
@@ -70,7 +70,7 @@ pkg install termux-api
 
 ## Building it!
 
-Open up your favorite editor, make a new file titled `myclip.go` (you can name it whatever you want, really) and add in
+Open up your favorite editor (I recommend VS Code and GoLand), make a new file titled `myclip.go` (you can name it whatever you want, really) and add in
```

Comment:
Maybe use or?

Reply:
But I recommend both lol. Here "and" is more appropriate.

Reply:
If I was saying "I recommend using..." then "or" would be better

Reply:
k